packages/jest-util/src/__tests__/buffered_console.test.js

```diff
@@ -0,0 +1,134 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+import chalk from 'chalk';
+import BufferedConsole from '../buffered_console';
+
+describe('CustomConsole', () => {
+  let _console;
+  const stdout = () =>
+    _console
+      .getBuffer()
+      .map(log => log.message)
+      .join('\n');
```

Comment:
you can use `strip-ansi` here to avoid having to use chalk inline. up to you 🙂

Reply:
Yes, maybe it's the way to go, but that way we don't test the **bold** effect.

WDYT?

Reply:
good point!